pages/home.html

```diff
@@ -100,10 +104,7 @@
     Contributing to Oppia
   </h2>
   <p>
-    Oppia is built by the community for the community. Our mission is to enable
-    anyone to learn anything they want to in an effective, enjoyable way, and
-    building a platform that allows anyone to create and share bite-sized
-    interactive learning material is just a first step.
+    Built by the community for the community, Oppia helps anyone to learn anything they want to in an effective, enjoyable way. Building a platform that allows people to create and share bite-sized interactive learning material is just a first step.
```

Comment:
This now feels like it's hanging. Perhaps:

Built by the community for the community, Oppia helps anyone to learn anything they want to in an effective, enjoyable way. So far, we've created a platform that allows people to create and share bite-sized interactive learning material, but this is just a first step.